docs/reference/release-notes/860.md

```diff
@@ -19,6 +19,71 @@ These release notes identify the new features included in 8.6, including [alpha
 | ---------------------- | ---------------------------- | ------------ | ------------ | --------------------------------------------------------------------------------------- |
 | 8 October 2024         | 14 April 2026                |              |              | [Self-Managed update guide](self-managed/operational-guides/update-guide/850-to-860.md) |
 
+### Migrate process management endpoints to Camunda 8 REST API
+
+<!-- https://github.com/camunda/product-hub/issues/2040 -->
+
+We're thrilled to announce the integration of Zeebe endpoints into Camunda 8 REST API, aimed at enhancing your experience in building process applications. This will ease the installation and onboarding to Camunda 8 as gRPC won't be required anymore. With the introduction of Zeebe endpoints in REST API, we're eliminating the need for gRPC, reducing friction in adoption. Developers can now seamlessly transition from gRPC to REST with the assurance of having the same endpoints, thereby saving time and effort. Use officially supported Camunda SDKs to have an easy migration journey.
+
+### Investigate and resolve incidents with execution listeners
+
+<!-- https://github.com/camunda/product-hub/issues/2103 -->
+
+Users can now easily check details of active and completed execution listeners for process activities and process instances. Operations engineers can efficiently resolve execution listener incidents. This enhancement streamlines operations and ensures smoother handling of incidents, ultimately improving time to unblock process execution.
+
+### Deprecate zbctl and GO client
+
+<!-- https://github.com/camunda/product-hub/issues/2438 -->
+
+With Camunda 8 REST API for Process Management, we will move the Go and zbctl clients to the Camunda Community Hub. This means that these clients won’t be officially supported by Camunda anymore.
+
+With the REST API, the interaction with the Zeebe engine will be much easier; you will be able to work with it in the same way you work with any other REST service. Use the OpenAPI definition, the supported Postman collection, or cURL for an easy getting-started experience and implement API calls in your code. We also recommend giving the new clients and SDKs a try.
+
+If you are interested in maintaining the community projects, contact us by filling out this form in the Camunda Community Hub!
```

Comment:
@aleksander-dytko @xomiamoore does this form exist? How should people contact us?

Reply:
It's the "Maintainer Proposal" template here -- https://github.com/Camunda-Community-Hub/community/issues/new/choose